openslides/motions/static/js/motions/diff.js

```diff
@@ -4,11 +4,13 @@
 
 angular.module('OpenSlidesApp.motions.diff', ['OpenSlidesApp.motions.lineNumbering'])
 
-.service('diffService', function (lineNumberingService) {
+.service('diffService', function (lineNumberingService, $cacheFactory) {
```

Comment:
Ah I think it is necessary to use the array notation:
```javascript
.service('diffService', [
    'lineNumberingService',
    '$cacheFactory',
    function (lineNumberingService, $cacheFactory) {
        ...
    }
])
```
This is required if we later want to compress and uglify the code via gulp.